### PR TITLE
docs: update starkgate_function_reference.adoc

### PR DESCRIPTION
**Pull request by xiaolou86**
### Description of the Changes

1. Update starkgate's references

2. Delete the L1 Constant `MIN_DEPLOYMENT_FEE`

### Check List

- [x] Changes have been done against main branch, and PR does not conflict
- [x] PR title follows the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1081)

### CODE REVIEW

**Comment by xiaolou86** [comment]
I can't find the L1 Constant MIN_DEPLOYMENT_FEE in any solidity, so I try to delete it...

Please tell me anything whether I am wrong or not.

File: components/Starknet/modules/tools/pages/starkgate_function_reference.adoc [hunk 1]
```diff
@@ -137,9 +137,9 @@ include::starkgate_function_reference.adoc[tag=uint256_l2Recipient]
 [discrete]
 ==== Function and event definitions
 
-Contract: link:https://github.com/starkware-libs/starkgate/blob/2f96bca50cc6203645b419a890b4a066d617a1cd/src/solidity/StarknetTokenBridge.sol[`StarknetTokenBridge.sol`]
+Contract: link:https://github.com/starknet-io/starkgate-contracts/tree/v2.0/src/solidity/StarknetTokenBridge.sol[`StarknetTokenBridge.sol`]
 
-* Function: link:https://github.com/starkware-libs/starkgate/blob/2f96bca50cc6203645b419a890b4a066d617a1cd/src/solidity/StarknetTokenBridge.sol#L290[`deposit`]
+* Function: link:https://github.com/starknet-io/starkgate-contracts/tree/v2.0/src/solidity/StarknetTokenBridge.sol#L284[`deposit`]
 * Event: link:[`Deposit`]
```
Comment (stoobie) [inline components/Starknet/modules/tools/pages/starkgate_function_reference.adoc:143]:
```suggestion
* Event: link:https://github.com/starknet-io/starkgate-contracts/blob/a4caa787263cb9a7cf01dbb7247957b586c6978a/src/solidity/StarknetTokenBridge.sol#L65[`Deposit`]
```

Comment (xiaolou86) [reply components/Starknet/modules/tools/pages/starkgate_function_reference.adoc]: Thank you very much, my bro @stoobie !

**Comment by xiaolou86** [comment]
Thank you very much, my bro @stoobie !

**Pull request merged**